code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -618,7 +618,7 @@
 		addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand(15,60))
 		qdel(holder.my_atom) //deleto
 	else
-		var/mob/living/simple_animal/slime/random/S = new (holder.my_atom.loc)
+		var/mob/living/simple_animal/slime/random/S = new (get_turf(holder.my_atom.loc))
```

Comment:
https://github.com/tgstation/tgstation/pull/35707/files/c94a08f90c8057eb38c760b397fb223aaf296d23#diff-42c31ae821efe57835573230af194682
@Cyberboss on suicide watch

Reply:
recommend that you change this to `get_turf(holder.my_atom)`, this skips an unneeded .loc. `my_atom` and `my_atom.loc` have the same turf

Reply:
Might want to make it drop_location as initially proposed so it doesn't move the newfound slime of mechs and closets

Reply:
Think I'll just make it `get_turf(holder.my_atom)`, since that's the behavior for most other slime reactions.